src/Catalog/Model/Source/LocalizedCatalogGroupOption.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Gally to newer versions in the future.
+ *
+ * @package   Gally
+ * @author    Gally Team <elasticsuite@smile.fr>
+ * @copyright 2022-present Smile
+ * @license   Open Software License v. 3.0 (OSL-3.0)
+ */
+
+declare(strict_types=1);
+
+namespace Gally\Catalog\Model\Source;
+
+use ApiPlatform\Core\Annotation\ApiProperty;
+use ApiPlatform\Core\Annotation\ApiResource;
+use Gally\Catalog\Model\Catalog;
+use Gally\Catalog\Model\LocalizedCatalog;
+
+#[ApiResource(
+    itemOperations: [],
+    collectionOperations: [
+        'get' => ['pagination_enabled' => false],
```

Comment:
I think this api is only the backoffice, may be we should restrict it the contributor user ?